deployment/build-and-stage.yaml

```diff
@@ -172,6 +172,10 @@ steps:
 - name: 'gcr.io/oss-vdb/deployment'
   args: ['bash', '-ex', 'gcp/appengine/deploy.sh', 'oss-vdb-test', 'deployment/gae/oss-vdb-test', 'app.yaml']
 
+# Clean up all but the last 5 App Engine versions to avoid hitting the ceiling on versions
```

Comment:
nit: Comment here still says 5

Reply:
Thanks fixed in [13bb8cc](https://github.com/google/osv.dev/pull/1191/commits/13bb8ccdadeba0cc0461e91d8babb767ab743b93)